src/test/scala/com/cognite/spark/datasource/TimeSeriesRelationTest.scala

```diff
@@ -33,6 +33,7 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest {
       .sql(s"""
          |select '$initialDescription' as description,
          |concat('TEST', name) as name,
+         |concat('TEST', name) as legacyName,
```

Comment:
Remove this from tests, we don't want to change the external schema and it doesn't look like we are. :+1: